controller/volume_controller.go

```diff
@@ -115,6 +117,7 @@ func NewVolumeController(
 		kubeClient:    kubeClient,
 		eventRecorder: eventBroadcaster.NewRecorder(scheme, v1.EventSource{Component: "longhorn-volume-controller"}),
 
+		nStoreSynced:  nodeInformer.Informer().HasSynced,
```

Comment:
your missing the wait for CacheSync on this one, below.

Reply:
Thanks, fixed.